Testing/VelaptorTests/Content/ContentLoaderTests.cs

```diff
@@ -87,92 +81,88 @@ public void Load_WhenLoadingFonts_LoadsFont()
             var loader = CreateContentLoader();
 
             // Act
-            loader.Load<IFont>(FontName);
+            loader.LoadFont(FontContentName, 12);
 
             // Assert
-            this.mockFontLoader.Verify(m => m.Load(FontName), Times.Once());
+            this.mockFontLoader.Verify(m => m.Load($"{FontContentName}|size:12"), Times.Once());
         }
 
         [Fact]
-        public void Load_WhenLoadingUnknownContent_ThrowsException()
+        public void Unload_WhenUnloadingTextures_UnloadsTexture()
```

Comment:
Refactor method name 
```suggestion
        public void UnloadTexture_WhenUnloadingTextures_UnloadsTexture()
```